src/components/FilterCard.vue

```diff
@@ -84,6 +105,7 @@ export default {
     async resetFilter() {
       this.filterValue = ''
```

Comment:
Maybe this should be an object instead? You have `filterValue` and `filterInput` and `filterWildcard` which would indicate to me that it should be `filter: { input: '', wildcard: '', value: '' }`. You would also then be able to make `getFilteredData` a method of the object. It would give better encapsulation

Reply:
Does that alter the efficiency of the UI?  Probably yes, but not enough to make it worth while avoiding this approach.  I don't know at what level the responsiveness of the UI operates, but if it updates UI elements because the `filter` object changed when the containing value (out of the three) hasn't changed, that could, on a larger scale, create a much larger overhead on the UI.  I doubt that's a factor here, but just wanted to identify a possible cost to what is being suggested.  It could be that Vue is far better at this than I realise.

Reply:
Since getFilteredData requires 'this' and access to the fetcher im not sure it should be included as a method of filter. We could import from a filterObject from another module but this would be a bit more complicated and require splitting it up, which would also make the code more cognitively difficult to follow.